src/views/preview/preview.jsx

```diff
@@ -513,8 +528,17 @@ const mapStateToProps = state => {
         userPresent;
     const authorPresent = projectInfoPresent && state.preview.projectInfo.author &&
         Object.keys(state.preview.projectInfo.author).length > 0;
+    const userOwnsProject = isLoggedIn && authorPresent &&
+        state.session.session.user.id === state.preview.projectInfo.author.id;
 
     return {
+        canAddToStudio: isLoggedIn && userOwnsProject,
+        canCreateNew: false,
+        canRemix: false,
+        canReport: isLoggedIn && !userOwnsProject,
+        canSave: userOwnsProject,
+        canSaveAsCopy: false,
+        canShare: true,
```

Comment:
Only if `userOwnsProject` (and if the user `isSocial`).

Reply:
agreed, but for later

Reply:
This one I think we can do now though. We have all of this information available. It would just make the new project page act more similar to the existing one.

Reply:
But I think this is fine to leave for now.

Reply:
I see your point. Changed to `canShare: userOwnsProject && state.permissions.social`

Reply:
Changed after discussion